### PR TITLE
[12.0][MIG] base_custom_info: Migration to 12.0

### PR DESCRIPTION
**Pull request by Tardo**
Cc @tecnativa TT18593

### CODE REVIEW

**Comment by AdriaGForgeFlow** [comment]
Are you planning on finishing this? Do you need help?

**Comment by pedrobaeza** [comment]
Yes, we need to check a problem with the onchanges. I'll check this week.

**Comment by AdriaGForgeFlow** [comment]
> Yes, we need to check a problem with the onchanges. I'll check this week.

Okay, thanks!

**Comment by pedrobaeza** [comment]
This should be now reviewable. @Tardo I think you can squash a bit commits.

**Comment by Tardo** [comment]
Ready for review

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@AdriaGForgeFlow can you review?

File: base_custom_info/README.rst [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+I.e., the "What weaknesses does he/she have?" *property* has some options:
+
+- Loves junk food
+- Needs videogames
+- Huge glasses
+
+The *value* will always be one of these.
+
+Recursive templates using options
```
Comment (AdriaGForgeFlow) [inline base_custom_info/README.rst]: Is this feature enabled? I don't seem to find how to use it.

**Review by AdriaGForgeFlow — CHANGES_REQUESTED** [review]
Done a code review and functional review and overall looks good to me.

Just a small error when trying to access the Values menu in Custom Info / Basic.
image

Another comment below.

File: base_custom_info/README.rst (hunk 1 above)
Comment (Tardo) [reply base_custom_info/README.rst]: I think that this refers to the feature of mix properties of different templates for a selection.

**Comment by Tardo** [comment]
Change done! thanks for the review

File: base_custom_info/README.rst (hunk 1 above)
Comment (AdriaGForgeFlow) [reply base_custom_info/README.rst]: Yes, but how do you use it? How do you specify in a selection property that links to a different template?
When I follow the instructions in the README and select "Needs videogames" in the Smart Partners template, nothing happens, no new properties appear in the list to be filled. Even though I see this in the property form view:
image

See:
image

Comment (Tardo) [reply base_custom_info/README.rst]: Finally, it's an outdated readme... moved to roadmap. This feature it's not implemented yet. Thanks!

**Comment by pedrobaeza** [comment]
Let's merge it then!

/ocabot merge

**Pull request merged**